gradio/cli.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import gradio.reload
+import gradio.upload
+
+def cli():
+    args = sys.argv[1:]
+    if len(args) == 0:
+        raise ValueError("No file specified.")
+    if args[0] == "--upload-to-spaces":
```

Comment:
Really a nit but I'd rather name it "--deploy". It's more concise (quicker to type) and implicitly make "spaces" the default way of deploying a Gradio app. But this is personal opinion so feel free to ignore.

Reply:
Agreed!